src/inject/utils/dom.ts

```diff
@@ -97,9 +97,10 @@ export function watchForNodePosition<T extends Node>(
             start = now;
             attempts = 1;
         }
-        if (prevSibling && prevSibling.parentNode !== parent) {
-            logWarn('Unable to restore node position: sibling was removed', node, prevSibling, parent);
-            stop();
+        if (prevSibling && (prevSibling.parentNode !== parent && prevSibling.parentNode !== null)) {
+            logWarn('Style was moved to another parent, moving node\'s position', node, prevSibling, parent);
+            prevSibling.parentNode.insertBefore(node, prevSibling.nextSibling);
```

Comment:
`parent` variable should be updated with the new sibling's parent.

Reply:
parent is a const outside the trothle and should be updated everytime a node is being changed

Reply:
So, use `let`. The goal of this function is to stick a node to some previous element. So, the `parent` should be changed only when it is changed.